Number_types/include/CGAL/boost_mp.h

```diff
@@ -607,6 +782,325 @@ namespace internal {
 #endif
 } // namespace internal
 
+#ifdef CGAL_USE_BOOST_MP
+
+template < >
+inline void
+simplify_quotient(boost::multiprecision::cpp_int & a, boost::multiprecision::cpp_int & b) {
```

Comment:
It is more common to overload than specialize, but here it seems ok. On the other hand, it would be possible to make it a bit more general so it also works with `number<cpp_int_backend<512>>` for instance.

Reply:
I changed it to the overload version.